one.lfa.android.services/src/main/java/one/lfa/android/services/LFAProfileModificationFragment.kt

```diff
@@ -519,6 +526,53 @@ class LFAProfileModificationFragment : ProfileModificationAbstractFragment() {
     }
   }
 
+  private fun updateProfileDefaultAccount(profileId: ProfileID) {
+    val profile = this.profilesController.profiles()[profileId]
+
+    if (profile != null) {
```

Comment:
In future, in order to have less indentation in code, it is better to check for null, log, and then exit the function early here.